frontend/public/components/factory/list-page.jsx

```diff
@@ -350,6 +353,7 @@ export const ListPage = props => {
 };
 
 ListPage.displayName = 'ListPage';
+export const ListPageWrapper = listPageWithFallback(ListPage);
```

Comment:
I don't know that we need another `ListPage` wrapper.

Reply:
removed the wrapper and moved the hoc to the list page const declaration